pages/add_customer_page.py

```diff
@@ -0,0 +1,41 @@
+import random
+
+from pages.locators import AddCustomerLocators
+from pages.base_page import BasePage
+
+
+class AddCustomerPage(BasePage):
+    @staticmethod
+    def generate_post_code(length=10):
+        return "".join(random.choices("0123456789", k=length))
+
+    @staticmethod
+    def generate_first_name(post_code):
+        numbers = [int(post_code[i:i + 2]) for i in range(0, len(post_code), 2)]
+
+        first_name = ""
+        for number in numbers:
+            first_name += chr((number % 26) + ord("a"))
+
+        return first_name
+
+    def fill_in_first_name_field(self, text):
+        input_first_name = self.browser.find_element(*AddCustomerLocators.First_Name)
+        input_first_name.send_keys(text)
```

Comment:
Здесь и далее. Если не планируем использовать переменную где-то ещё, то можно сократить код.
```suggestion
    def fill_in_first_name_field(self, text):
        self.browser.find_element(*AddCustomerLocators.First_Name).send_keys(text)
```